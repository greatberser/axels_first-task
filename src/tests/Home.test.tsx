import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

describe('Home', () => {
  it('should match a snapshot', () => {
    const { asFragment } = render(<div>Home Component</div>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('should contain Home navigation link', () => {
    render(
      <nav>
        <a href="/">Home</a>
        <a href="/form">Form</a>
      </nav>
    );
    expect(screen.getByText(/home/i)).toBeInTheDocument();
  });

  it('should contain Form navigation link', () => {
    render(
      <nav>
        <a href="/">Home</a>
        <a href="/form">Form</a>
      </nav>
    );
    expect(screen.getByText(/form/i)).toBeInTheDocument();
  });
});
