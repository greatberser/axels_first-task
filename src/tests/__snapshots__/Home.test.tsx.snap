// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Home should match a snapshot 1`] = `
<DocumentFragment>
  <div>
    Home Component
  </div>
</DocumentFragment>
`;
